services/openvpn/client.go

```diff
@@ -131,7 +131,7 @@ type VPNConfig struct {
 	OriginalRemoteIP   string
 	OriginalRemotePort int
 
-	DNS             string `json:"dns"`
+	DNS             string `json:"dns_servers"`
```

Comment:
Maybe rename this field to DNSServers too to match json field.  It would be easer to understand because usually you expect json structure to be the same as your fields names.